src/components/common/icons/base.js

```diff
@@ -51,15 +51,18 @@ export default class Base extends Component {
     width: null,
     viewBox: '0 0 64 64',
     predefinedClassName: '',
-    className: ''
+    className: '',
```

Comment:
setting `fill: 'currentColor'` here means if user pass in a style object, it will override `fill` prop. use `style.fill = 'currentColor';` in render simply adds it to user defined value

Reply:
The problem with style.fill = 'currentColor' is that i cannot override the default value;
I need a way to override the fill property.

we can do something like
```
style = {}
... } = this.props;

style.fill = style.fill || 'currentColor'
```

or I can create a helper to merge defaults values when no properties exist in the original object